### DISCUSSION TITLE
[v3] Script tags without `data-navigate-*` throw errors when using `wire:navigate`

### DISCUSSION THREAD

**Discussion by jacksleight** [livewire/livewire#5882] [7. Bugs (archived)] [answered]
When using `wire:navigate` to navigate to a page with a script tag that has attributes and none of those attributes are `data-navigate-*`. eg:

```js
<script type="application/ld+json">
{
    "@context": "https://schema.org/",
    "@type": "Organization"
}
</script>
```

An error is thrown before the DOM is updated:

![Screenshot 2023-07-22 at 16 39 45](https://github.com/livewire/livewire/assets/126740/1fbaabab-54e6-4d4a-b3d7-3d9bd623a395)

Using v3.0.0-beta.3.

**Comment by joshhanley** [comment]
@jacksleight is it working for a normal script tag for you? Just want to confirm if it's the JSON-LD type causing issues.

**Reply by jacksleight** [reply]
Ah so turns out it's not actually anything to do with the JSON-LD type or even the `type` attribute specifically. I'm seeing that error whenever:

* There's a script tag in the `head` that has attributes and none of those attributes are `data-navigate-track`
* There's a script tag in the `body` that has attributes and none of those attributes are `data-navigate-once`

So in the head:

* `<script></script>` - works
* `<script type="application/ld+json"></script>` - errors
* `<script type="application/ld+json" data-navigate-track></script>` - works
* `<script type="application/ld+json" data-navigate-once></script>` - errors

And in the body:

* `<script></script>` - works
* `<script type="application/ld+json"></script>` - errors
* `<script type="application/ld+json" data-navigate-once></script>` - works
* `<script type="application/ld+json" data-navigate-track></script>` - errors

**Reply by jacksleight** [reply]
I've done a bit more testing and this only seems to be happening when manually bundling Livewire and using `@livewireScriptConfig`. Injecting the scripts automatically or using `@livewireScripts` works fine.

I've created a demo repo that shows the issue: https://github.com/jacksleight/sandbox-livewire/tree/main

**Reply by riasvdv** [reply]
Someone else has done a bug report on the Alpine side: https://github.com/alpinejs/alpine/discussions/3671

**Comment by joshhanley** [comment, marked as answer]
@jacksleight the `arr is not defined` error has been fixed in https://github.com/alpinejs/alpine/pull/3685 and will be in the next beta release. Can you test it again once it has been tagged and let us know if it fixes your issue? Thanks!

**Reply by jacksleight** [reply]
Amazing, thanks! Will do.

**Reply by jacksleight** [reply]
Tested with v3.0.0-beta.8 and working perfectly!
